cmake/CMakeLists.txt

```diff
@@ -53,11 +53,18 @@ include(CheckIncludeFileCXX)
 
 if(${CMAKE_CXX_COMPILER_ID} STREQUAL "Intel")
   set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -restrict")
+  if (NOT CMAKE_TUNE_FLAGS)
```

Comment:
Shouldn't there is an `option(CMAKE_TUNE_FLAGS "Documentation String" OFF)` to document this?

Reply:
Or a cached string with documentation.

Reply:
Will look into it. You know CMake so much better than me, that is why I am asking for feedback.

Reply:
I think `set(CMAKE_TUNE_FLAGS "" CACHE STRING "Tune flags")` is what we want.

Reply:
I'm going for a cached string variable which has an internal default.